packages/ion-did-provider/src/ion-did-resolver.ts

```diff
@@ -0,0 +1,7 @@
+import * as ION from '@sphereon/ion-tools'
+
+export function IonDIDResolver() {
```

Comment:
Hmzz, this should probably be more inline with the Resolver package from DIF, conforming to the Interface for resolution. Similar to what we have in the did-uni-client. Veramo also supports that interface for DID resolution

Reply:
did some refactoring on the ion resolver

Reply:
LGTM